docs/source/qvm.rst

```diff
@@ -36,43 +52,242 @@ QVM server by typing ``qvm -S`` into your terminal. You should see the following
+run your program, a pyQuil client will send a Quil program to the QVM server and wait for a response back.
+
+It's also possible to use the QVM from the command line. You can write a Quil program in its own file:
+
+.. code:: python
+
+    # example.quil
+
+    DECLARE ro BIT[1]
+    RX(pi/2) 0
+    CZ 0 1
+
+and then execute it with the QVM directly from the command line:
+
+.. code:: python
+
+    $ qvm -e < example.quil
+
+    [2018-11-30 11:13:58] Reading program.
+    [2018-11-30 11:13:58] Allocating memory for QVM of 2 qubits.
+    [2018-11-30 11:13:58] Allocation completed in 4 ms.
+    [2018-11-30 11:13:58] Loading quantum program.
+    [2018-11-30 11:13:58] Executing quantum program.
+    [2018-11-30 11:13:58] Execution completed in 6 ms.
+    [2018-11-30 11:13:58] Printing 2-qubit state.
+    [2018-11-30 11:13:58] Amplitudes:
+    [2018-11-30 11:13:58]   |00>: 0.0, P=  0.0%
+    [2018-11-30 11:13:58]   |01>: 0.0-1.0i, P=100.0%
+    [2018-11-30 11:13:58]   |10>: 0.0, P=  0.0%
+    [2018-11-30 11:13:58]   |11>: 0.0, P=  0.0%
+    [2018-11-30 11:13:58] Classical memory (low -> high indexes):
+    [2018-11-30 11:13:58]     ro:  1 0
+
+For a detailed description of how to use the ``qvm`` from the command line, see :ref:`The QVM manual page <qvm_man>` or
+type ``man qvm`` in your terminal.
+
+We also offer a Wavefunction Simulator (formerly a part of the :py:class:`~pyquil.api.QVM` object),
+which allows users to contruct and inspect wavefunctions of quantum programs. Learn more
+about the Wavefunction Simulator :ref:`here <wavefunction_simulator>`.
+
+.. _qpu:
+
+The Quantum Processing Unit (QPU)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To access a QPU endpoint, you will have to `sign up <https://www.rigetti.com/>`_ for Quantum Cloud Services (QCS).
+Documentation for getting started with your Quantum Machine Image (QMI) is found
+`here <https://www.rigetti.com/qcs/docs/intro-to-qcs>`_. Using QCS, you will ``ssh`` into your QMI, and reserve a
+QPU lattice for a particular time block.
+
+When your reservation begins, you will be authorized to access the QPU. A configuration file will be automatically
+populated for you with the proper QPU endpoint for your reservation. Both your QMI and the QPU are located on premises,
+giving you low latency access to the QPU server. That server accepts jobs in the form of ``BinaryExecutableRequest``s,
+which is precisely what you get back when you compile your program in pyQuil and target the QPU (more on this soon).
+This request contains all the information necessary to run your program on the control rack which sends and receives
+waveforms from the QPU, so that you can receive classified readout results (``0``s and ``1``s).
+
+For information on available lattices, you can check out your dashboard at https://qcs.rigetti.com/dashboard after you've
+been invited to QCS.
+
+
+.. _quantum_computer:
+
+The ``QuantumComputer``
```

Comment:
Would consider adding a part on getting current QPU performance post-retune in pyQuil (somewhere in here). Might be helpful as users consider what lattice they want to use

Reply:
How do I do that as a user? @mpharrigan